### PR TITLE
PLATFORM-3966 legacy domains

### PR DESCRIPTION
**Pull request by harnash**
Properly set alternative domain for non english language wikis created on fandom.com domain.

### CODE REVIEW

File: maintenance/wikia/HttpsMigration/fixWikiFactoryDomains.php [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+				"updating WF domain for wiki: {$this->COLOR_GREEN}$originalDomain{$this->COLOR_NONE} -> " .
+				"{$this->COLOR_CYAN}$finalDomain{$this->COLOR_NONE}: "
+			);
+			if ( empty( $finalDomain) ) {
+				$this->output( "no change computed for wiki - {$this->COLOR_CYAN}skipping{$this->COLOR_NONE}\n" );
+				continue;
+			}
+
+			if ( $saveChanges ) {
+				if ( !WikiFactory::removeDomain( $cityId, $originalDomain, [ 'fixing legacy domain after fandom.com migration' ] ) ) {
```
Comment (Grunny) [inline maintenance/wikia/HttpsMigration/fixWikiFactoryDomains.php]: The reason parameter should just be a string and not an array, right?

File: maintenance/wikia/HttpsMigration/fixWikiFactoryDomains.php [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+					$this->output( "failed to remove original domain: {$this->COLOR_RED}$originalDomain{$this->COLOR_NONE}\n" );
+					continue;
+				}
+				if ( !WikiFactory::addDomain( $cityId, $finalDomain, [ 'fixing legacy domain after fandom.com migration' ] ) ) {
```
Comment (Grunny) [inline maintenance/wikia/HttpsMigration/fixWikiFactoryDomains.php]: Same here.

**Review by Grunny — CHANGES_REQUESTED** [review]

File: maintenance/wikia/HttpsMigration/fixWikiFactoryDomains.php (hunk 1 above)
Comment (harnash) [reply maintenance/wikia/HttpsMigration/fixWikiFactoryDomains.php]: Right my bad

Comment (harnash) [reply maintenance/wikia/HttpsMigration/fixWikiFactoryDomains.php]: Fix in e5c5312763f

File: maintenance/wikia/HttpsMigration/fixWikiFactoryDomains.php (hunk 2 above)
Comment (harnash) [reply maintenance/wikia/HttpsMigration/fixWikiFactoryDomains.php]: Yup: e5c5312763f

**Review by Grunny — APPROVED** [review]

**Pull request merged**